### PR TITLE
Expect OMPFLAGS from environment

### PR DESCRIPTION
**Pull request by mahrud**
I'm not sure if there's a way to set a default value for OMPFLAGS, but if that's possible, feel free to push to this branch to do that instead. I just wanted to provide one option for #309.

### CODE REVIEW

File: fflas-ffpack/fflas/fflas_sparse/read_sparse.h [hunk 1]
```diff
@@ -447,8 +447,8 @@ namespace FFLAS {
         file.write((char*) &magic     , sizeof(int64_t)) ;
         file.write((char*) &field_desc, sizeof(int64_t)) ;
         file.write((char*) &data_type , sizeof(int64_t)) ;
-        file.write((char*) &rowdim , sizeof(int64_t)) ;
-        file.write((char*) &coldim , sizeof(int64_t)) ;
+        file.write((const char*) &rowdim , sizeof(int64_t)) ;
```
Comment (ClementPernet) [inline fflas-ffpack/fflas/fflas_sparse/read_sparse.h:450]: These edits are unrelated with this PR. Can you remove them, and possibly open another one ? Thanks

**Comment by mahrud** [comment]
Not soon. Feel free to push to my branch and excise that commit, or just
make the omp flags change in a separate commit and close this PR.

**Comment by ClementPernet** [comment]
I've removed the commit and made this #332 hence I'm closing this PR.
